applications/nrf5340_audio/README.rst

```diff
@@ -352,6 +352,11 @@ The first part of these names is based on :kconfig:option:`CONFIG_BT_DEVICE_NAME
 
 .. _nrf53_audio_app_building:
 
+Note
```

Comment:
This note should be above ".. _nrf53_audio_app_building:"

Reply:
Good catch.